admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/services/AdminService.scala

```diff
@@ -17,12 +19,39 @@ class AdminService @Inject() (
 ) {
 
   def tables(): Future[AdminGetTables.Response] = {
-    Future.successful {
-      val items = tableSettings.tables.map { x =>
-        AdminGetTables.Response.DatabaseTable(name = x.tableName)
-      }
-      AdminGetTables.Response(items)
+    def getFieldName(fieldName: String, primaryKeyField: String) = {
+      val isPrimaryField = fieldName == primaryKeyField
+      if (isPrimaryField) "id" else fieldName
```

Comment:
We are missing a comment like `// NOTE: react-admin requires the id field to be available`